owncloud/core
# federated sharing auto-complete, first step

This is the first part of the federated sharing auto-complete feature for ownCloud 9.0 (https://github.com/owncloud/core/issues/19566)

It provides a user interface for the admin to add and remove trusted ownClouds.

Steps to test:
- Enable the app "federation"
- Go to the admin settings
- You should be able to add/remove ownCloud servers from the list of trusted servers

In the next step we will exchange the user list between trusted ownClouds and use it in the share dialog for auto-complete
